### PR TITLE
docs: update angular alternatives

### PR DESCRIPTION
**Pull request by sympatheticmoose**
**What is this feature?**

Updates with some more alternatives for HTML and SVG plugins

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/developers/angular_deprecation/angular-plugins.md [hunk 1]
```diff
@@ -680,7 +680,7 @@ Lack of recent activity in the [project repository](https://github.com/MarcusCal
 
 > **Migration available - potential alternative:** Grafana provides the native [Canvas]({{< relref "../../panels-visualizations/visualizations/canvas/" >}}) panel.
 
-> **Migration available - potential alternative:** another plugin exists which provides similar capabilities - [ACE.SVG](https://grafana.com/grafana/plugins/aceiot-svg-panel/). Note this panel has compatibility issues with Grafana versions 10.0.0-10.1.0.
+> **Migration available - potential alternative:** other plugins exist which provide similar capabilities - [HTML](https://grafana.com/grafana/plugins/gapit-htmlgraphics-panel/), [Dynamic Text](https://grafana.com/grafana/plugins/marcusolsson-dynamictext-panel/) and [ACE.SVG](https://grafana.com/grafana/plugins/aceiot-svg-panel/) - note this panel has compatibility issues with Grafana versions 10.0.0-10.1.0.
```
Comment (josmperez) [inline docs/sources/developers/angular_deprecation/angular-plugins.md:683]:
```suggestion
> **Migration available - potential alternative:** other plugins exist which provide similar capabilities - [HTML](https://grafana.com/grafana/plugins/gapit-htmlgraphics-panel/), [Dynamic Text](https://grafana.com/grafana/plugins/marcusolsson-dynamictext-panel/) and [ACE.SVG](https://grafana.com/grafana/plugins/aceiot-svg-panel/). Note this panel has compatibility issues with Grafana versions 10.0.0-10.1.0.
```

**Review by josmperez — APPROVED** [review]
LGTM.

File: docs/sources/developers/angular_deprecation/angular-plugins.md (hunk 1 above)
Comment (sympatheticmoose) [reply docs/sources/developers/angular_deprecation/angular-plugins.md:683]: @josmperez the compatibility issue only applies to ACE.SVG, with this suggestion I think that becomes less clear. Any other alternatives?

Comment (josmperez) [reply docs/sources/developers/angular_deprecation/angular-plugins.md:683]: updated suggestion

**Pull request merged**